polling_stations/apps/data_collection/data_types.py

```diff
@@ -191,6 +192,17 @@ def get_uprn_lookup(self):
 
         return uprn_lookup
 
+    def get_postcode_lookup(self):
```

Comment:
Not tested this but would a `collections.Counter` object here remove the need for this method and the `is_split_postcode` function below?

```
c = Counter([r["postcode"] for r in self.elements)
c[record["postcode"]] > 1
```

Not really a functional PR message, but might reduce the number of lines and connotative load.

Reply:
I think the snippet you've given there would just count the number of times each postcode occurrs (which you would expect to be lots for every postcode, as it will occur once for each relevant address in the list) and doesn't take account of how many different polling stations its associated with (which is the thing we care about). That said, you're probably right that there is probably a niftier way of leveraging collections to do this job (and what I'm doing in `get_address_lookup()` / `get_uprn_lookup()` ). I'll have a think about it..